wagtail/admin/edit_handlers.py

```diff
@@ -844,3 +845,38 @@ def id_for_label(self):
     def on_model_bound(self):
         super().on_model_bound()
         self.block_def = self.db_field.stream_block
+
+
+class ReadOnlyPanel(EditHandler):
+    def __init__(self, attr, *args, **kwargs):
+        self.attr = attr
+        super().__init__(*args, **kwargs)
+
+    def clone(self):
+        return self.__class__(
+            attr=self.attr,
+            heading=self.heading,
+            classname=self.classname,
+            help_text=self.help_text,
+        )
+
+    def render(self):
+        value = getattr(self.instance, self.attr)
+        if callable(value):
+            value = value()
+        return format_html('<div style="padding-top: 1.2em;">{}</div>', value)
```

Comment:
Can you please move styling a more relevant place?

Reply:
@zerolab what is a specific suggestion?

Reply:
Are there any CSS helper classes that could be used here instead?

Reply:
A tough one. I would add it as an utility class under `client/scss/overrides/_utilities.legacy.scss`

Reply:
What about just removing the style for now?

```suggestion
        return format_html('<div>{}</div>', value)
```

Reply:
@thibaudcolas Would you be able to suggest the best place to add a style for this, and a sensible class name? It's a while since I've done anything with static files in Wagtail.